### PR TITLE
Add gethubwetfin's pin to the map

### PR DESCRIPTION
**Pull request by wetfin**
Adding vizag to the map!

### CODE REVIEW

File: _pins/gethubstudent.wetfin [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+gethubhandle:gethubwetfin
```
Comment (brianamarie) [inline _pins/gethubstudent.wetfin]: Hi @wetfin! Make sure the spelling is 'githubHandle', and make sure there is a space between the colon and the name for the githubHandle, latitude, and longitude. 

Let me know if you need any help!

**Comment by wetfin** [comment]
Thanks I will check it out.

File: _pins/gethubstudent.wetfin (hunk 1 above)
Comment (wetfin) [inline _pins/gethubstudent.wetfin]: Where do I rectify this error, in the githubHandle. I am not able to access that code

Comment (brianamarie) [inline _pins/gethubstudent.wetfin]: Hi @wetfin! To fix the code, click the 'files changed' tab near the top of the pull request, and then click the small grey pencil on the right side of your file.

**Comment by crichID** [comment]
@wetfin I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/2459/commits/f27a9576c1a2939a948c9d8867a669fca52851b3. Please take a quick look at the changes. Once the tests have finished running, please feel free to click the big green **Merge pull request** button. If you haven't merged it by Friday (January 27, 2017), I will go ahead and do it just to keep the repository clean!

**Pull request merged**